### PR TITLE
Native file name extraction for FILEDELETE

### PR DESCRIPTION
**Pull request by v-p-b**
A small change to natively extract filenames as requested in https://github.com/tklengyel/drakvuf/issues/98. Output will be in the following format (CSV and default):

```
filedelete,0,0x4d2f1a0,svchost.exe,"\Windows\Prefetch\TASKHOST.EXE-7238F31D.pf"
[FILEDELETE] VCPU:0 CR3:0x4d2f2e0,explorer.exe "\Users\b\Desktop\Új szöveges dokumentum.txt"
```

As you can see funny Hungarian characters and spaces in paths are handled (although I'm not sure about if CSV parsers treat `\` as escape character inside quotes...)

### CODE REVIEW

File: src/plugins/filedelete/filedelete.cpp [hunk 1]
```diff
@@ -206,7 +207,7 @@ static void grab_file_by_handle(filedelete *f, drakvuf_t drakvuf,
     vmi_read_16(vmi, &ctx, &length);
 
     if (length && buffer) {
-
+        char *procname = drakvuf_get_current_process_name(drakvuf, info->vcpu, info->regs);
```
Comment (tklengyel) [inline src/plugins/filedelete/filedelete.cpp]: This should get freed somewhere.

**Comment by tklengyel** [comment]
This is ok to test.

**Comment by v-p-b** [comment]
So maybe I fckd up something with Git or GitHub updated something but I don't see my latest commit to cause Jenkins to retest the PR. 

Anyway: I added the missing free, and moved the process name extraction inside the success branch so maybe we can spare a call now and then.

**Comment by tklengyel** [comment]
Nah, it's just that the original PR tester machine is offline and the one I setup for the interim is not online all the time. I just have to turn it on from time to time to do PR testing ;)

Retest this please.

**Pull request merged**